docs/streams/developer-guide/config-streams.html

```diff
@@ -63,24 +63,29 @@
           </ul>
           </li>
           <li><a class="reference internal" href="#optional-configuration-parameters" id="id6">Optional configuration parameters</a><ul>
+            <li><a class="reference internal" href="#acceptable-recovery-lag" id="id27">acceptable.recovery.lag</a></li>
             <li><a class="reference internal" href="#default-deserialization-exception-handler" id="id7">default.deserialization.exception.handler</a></li>
             <li><a class="reference internal" href="#default-production-exception-handler" id="id24">default.production.exception.handler</a></li>
             <li><a class="reference internal" href="#default-key-serde" id="id8">default.key.serde</a></li>
             <li><a class="reference internal" href="#default-value-serde" id="id9">default.value.serde</a></li>
+            <li><a class="reference internal" href="#max-task-idle-ms" id="id28">max.task.idle.ms</a></li>
```

Comment:
No wonder we get so many questions about this, the config has been missing from the docs the whole time

Reply:
Huh, oh yeah. I wonder if these are the docs that are autogenerated? Not sure, since it seems to still be missing some configs, but only configs that don't have an associated `{CONFIG_NAME}_DOC` string...

Anyways, I totally agree we should not have this listed in two places, but imo we should continue to list them in the [Streams docs](https://kafka.apache.org/documentation/streams/developer-guide/config-streams.html) where we can go into much more detail, and where users are more likely to look first. Personally, I forgot that we also listed configs here and only ever looked at the Streams page. I'd imagine I'm not alone in that.

Maybe we can remove the listing here and just link to the Streams config page?

Reply:
Not necessarily as part of this PR. I guess we can discuss what to do going forward; for now, I think it's best to just make sure both pages reflect the current reality